### DISCUSSION TITLE
pip3 install failed in net8 docker image.

### DISCUSSION THREAD

**Discussion by Yun-Ting** [dotnet/dotnet-docker#4949] [Q&A]
The image downloaded from mcr.microsoft.com/dotnet/sdk:net8.0
have issues when pip3 installation:
![image](https://github.com/dotnet/dotnet-docker/assets/16979322/b9cdf101-ed05-475e-8d12-2f9ccb136297)

What would be the suggested way to workaround this issue if I need the package in the CI flow?

Scenario: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4944
Thank you.

**Comment by richlander** [comment]
Debian is telling you that Python is not installed and which package to use to install it.

This will work:

```bash
RUN apt update && apt install -y python3-full
```

**Comment by cezarartsound** [comment]
It seems it works with `mcr.microsoft.com/dotnet/aspnet:8.0-jammy`. 🤗
